src/lib.rs

```diff
@@ -24,7 +24,8 @@
 //! | VxWorks           | `*‑wrs‑vxworks‑*`  | `randABytes` after checking entropy pool initialization with `randSecure`
 //! | ESP-IDF           | `*‑espidf`         | [`esp_fill_random`]
 //! | Emscripten        | `*‑emscripten`     | [`getentropy`][13]
-//! | WASI              | `wasm32‑wasi`      | [`random_get`]
+//! | WASI preview 1    | `wasm32‑wasip1`    | [`random_get`]
+//! | WASI preview 2    | `wasm32‑wasip2`    | [`get-random-u64`]
```

Comment:
Minor nit: can we call this "WASI 0.1" and "WASI 0.2" instead? That's the terminology we're encouraging people to use everywhere now, because we noticed the "preview" moniker can sometimes feel a little overwhelming.

```suggestion
//! | WASI 0.1    | `wasm32‑wasip1`    | [`random_get`]
//! | WASI 0.2    | `wasm32‑wasip2`    | [`get-random-u64`]
```